angular/demo/daisyui/src/app/samples/rating/rating.component.html

```diff
@@ -5,12 +5,11 @@
 			(mouseleave)="_widget.actions.leave()"
 			(mouseenter)="_widget.actions.hover(index + 1)"
 			(click)="_widget.actions.click(index + 1)"
-			[style.cursor]="state().interactive ? 'pointer' : 'default'"
```

Comment:
As we do not allow to update the readonly or disabled flag in this sample, the widget is always interactive.